packages/assets/tutorial/README.md

```diff
@@ -0,0 +1,239 @@
+
+![Add Plain Axes tool](./images/tutorial-12.png)
+
+The first node required is the root node. By default this should be named `<profileId>_<handedness>`, where any dashes (`-`) in the profileId are replaced with underscores (`_`). So because I'm preparing a `right` handed asset for the `oculus-touch` profile the root node's name will be `oculus_touch_right`, which is what the newly created axis (named `Empty` by default) should be renamed to in the outliner.
+
+![Renaming the newly created node](./images/tutorial-13.png)
+
+Finally, all of the meshes should be moved under the root node by selecting them in the outliner and dragging them on top of the new root node while holding down Shift.
+
+![Reparenting the meshes under the new node](./images/tutorial-14.png)
+
+Next, we'll set up a series of nodes that describe the motion extents of each component, which is easiest to describe by component type.
+
+## Buttons
+
+If your input device has any simple buttons like the "A" and "B" buttons on the Oculus Touch controller, they tend to be the easiest component to rig. Each moving component needs at minimum three nodes to describe it's motion. A "value", a "min", and a "max". At runtime the value node will be interpolated between the min and the max nodes, based on the value reported by the XRInputSource's gamepad. By extension, any meshes underneath that node in the scene hierarchy will be moved as well, creating our desired animation.
+
+The nodes must be named specifically so that the library can locate them and associate them with the appropriate gamepad inputs. (These names can be overriden if needed in the `profile.json`, but it's typically easiest to just use the default names.)
+
+For a simple button component the default node names are `<buttonName>_pressed_value`, `<buttonName>_pressed_min`, and `<buttonName>_pressed_max` where any dashes (`-`) in the buttonName are replaced with underscores (`_`). To make things easier, the profile viewer page from earlier will also tell you some of the node names it's unable to find. In my case the Oculus Touch's "A" will need a `a_button_pressed_value`, `a_button_pressed_min`, `a_button_pressed_max`.
+
+I typically find it easiest to create the value node first and position it. While it's not required, it's helpful to not only align the value node with the button's center, but also rotate it so that the axis points stright up out of the button.
+
+![Aligning a node with a button](./images/tutorial-15.png)
+![Adjusting the node orientation](./images/tutorial-16.png)
+
+Once positioned, right click and copy the node in the outliner, then paste two copies and rename them to the "min" and "max" variants respectively, which gives us all three nodes at the same place. In the outliner move those nodes under the "root" node, then drag the mesh for the corresponding component and place it under the "max" node.
```

Comment:
I think you should tell the user to delete the mesh from the copy and pasted min and max node, and that you will temporarily put the orignal geometry in it for testing.

Or perhaps position it with the geometry, then delete the geometry?

Reply:
Thanks, I've tweaked the language to try and make the operations performed on the mesh a bit clearer.